### PR TITLE
WanConsumerConfig code coverage

### PR DESCRIPTION
**Pull request by emrahkocaman**

### CODE REVIEW

File: hazelcast-spring/src/test/java/com/hazelcast/spring/TestFullApplicationContext.java [hunk 1]
```diff
@@ -636,6 +636,12 @@ public void testWanReplicationConfig() {
         assertEquals("com.hazelcast.wan.custom.WanConsumer", consumerConfig.getClassName());
         Map<String, Comparable> consumerProps = consumerConfig.getProperties();
         assertEquals("prop.consumer", consumerProps.get("custom.prop.consumer"));
+
+        WanReplicationConfig config2 = config.getWanReplicationConfig("testWan2");
+        WanConsumerConfig consumerConfig2 = config2.getWanConsumerConfig();
+        consumerConfig2.setProperties(consumerProps);
+        assertTrue(consumerConfig2.getImplementation() instanceof DummyWanConsumer);
```
Comment (pveentjer) [inline hazelcast-spring/src/test/java/com/hazelcast/spring/TestFullApplicationContext.java]: there is an assertInstanceOf

File: hazelcast-spring/src/test/java/com/hazelcast/spring/DummyWanConsumer.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.hazelcast.spring;
+
+public class DummyWanConsumer {
```
Comment (pveentjer) [inline hazelcast-spring/src/test/java/com/hazelcast/spring/DummyWanConsumer.java]: Is there no need to implement a certain interface?

Comment (emrahkocaman) [inline hazelcast-spring/src/test/java/com/hazelcast/spring/DummyWanConsumer.java]: Interface is in EE.

**Comment by serkan-ozal** [comment]
:+1:

**Comment by bilalyasar** [comment]
:+1:

**Pull request merged**